aws/storagemanager/spec/Makefile

```diff
@@ -0,0 +1,4 @@
+gen:
```

Comment:
we should probably run the gen rule in travis and fail if the aws.yaml generated is different from the one checked in. Can be in a followup PR.

Reply:
good idea. Will do.